src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -186,11 +194,15 @@
                                                :from 0
                                                :size 10}))))}
         :typeaheadDisplay (fn [result]
-                            ;; we intentionally do not encode the result value here, because it is already
-                            ;; encoded from the server.
-                            (.text (js/$ (str "<div>" (aget result "value") "</div>"))))
+                            (let [sugg-object (js->clj (aget result "value"))
+                                  suggestion (get sugg-object "suggestion")]
+                              (.text (js/$ (str "<div>" suggestion "</div>")))))
```

Comment:
I can't really tell how this particular value is used, but it seems to me the returned string needs to be sanitized. For example:
```javascript
$('<div><script>bad</script></div>').text() // -> 'bad'
```
but I can just do this:
```javascript
$('<div>&lt;script&gt;bad&lt;/script&gt;</div>').text() // -> '<script>bad</script>'
```

If the user of this value will be escaping it, then just a plain `suggestion` should be sufficient.